int32 box_id
int32 top_left_x
int32 top_left_y
int32 bottom_right_x
int32 bottom_right_y

